lib/pedicel/base.rb

```diff
@@ -43,40 +47,30 @@ def application_data
       [@token['applicationData']].pack('H*')
     end
 
-    def signing_time_ok?(now: Time.now)
-      # "Inspect the CMS signing time of the signature, as defined by section 11.3
-      # of RFC 5652. If the time signature and the transaction time differ by more
-      # than a few minutes, it's possible that the token is a replay attack."
-      # https://developer.apple.com/library/content/documentation/PassKit/Reference/PaymentTokenJSON/PaymentTokenJSON.html
-
-      delta = Pedicel.config[:replay_threshold_seconds]
-
-      cms_signing_time.between?(now - delta, now + delta)
-      # Deliberately ignoring leap seconds.
-    end
-
     def private_key_class
-      {EC_v1: OpenSSL::PKey::EC, RSA_v1: OpenSSL::PKey::RSA}[version]
+      { EC_v1: OpenSSL::PKey::EC, RSA_v1: OpenSSL::PKey::RSA }[version]
```

Comment:
The keys `:EC_v1` and `:RSA_v1` are symbols, but `version` appears to return a string.

Reply:
Version currently returns a string:
```ruby
    def version
      @token['version']&.to_sym
    end
```
This is a problem, however, since it is used in a string above:
```
      raise VersionError, "unsupported version: #{version}" unless SUPPORTED_VERSIONS.include?(@token['version'])
```
I'll change to consistent symbol use unless the other is preferred :)

Reply:
Why is that a problem?
```
irb(main):001:0> "#{:foobar}"
=> "foobar"
```

Reply:
Also `version` returns a *symbol* 🙂

Reply:
Hmm, when I reviewed and searched for "def version" on the page https://github.com/clearhaus/pedicel/pull/7/files , I got the following:

![screenshot_2018-04-20_13-58-47](https://user-images.githubusercontent.com/14069221/39049753-10b3a524-44a3-11e8-9075-1eae5e0bbc25.png)

i.e. `[@token['applicationData']].pack('H*')`. I still do. Weird. I have checked out the branch locally and see `@token['version']&.to_sym`.

My point was:

```
> { EC_v1: 1, RSA_v1: 2 }['EC_v1']=> nil
```

Reply:
Never mind :slightly_smiling_face: Notice on the screenshot that `def version` is on line 24, whereas `[@token['applicationData']].pack('H*')` is on line 47. GitHub collapsed the lines in between. Thanks to @ct-clearhaus for noticing.

Reply:
I was apparently cheated too. First after thinking why `applicationData` had to do with `version` my eyes could see 🙂

Reply:
I definitely meant symbol, but wrote string :)

It turns out to not be a problem, but I got sidetracked and looked at different issues here.
I had simply assumed that:
```ruby
puts("Hello " + :world)
```
would be functionally similar to:
```ruby
puts("Hello #{:world}")
```